sos/plugins/memory.py

```diff
@@ -37,7 +37,7 @@ def setup(self):
         self.add_cmd_output("free", root_symlink="free")
         self.add_cmd_output([
             "free -m",
-            "swapon --bytes --show"
+            "swapon --summary --verbose"
```

Comment:
Well, for info, this version was merged in Debian in july 2014, so after trusty's release.

And I don't expect anything done upstream for this as this has now become historical. Easier for me to just carry a simple distro patch for Trusty only. I just felt it was worth mentionning